extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -74,6 +75,11 @@ export class AmpAd extends AMP.BaseElement {
       const slotId = window.ampAdSlotIdCounter++;
       this.element.setAttribute('data-amp-slot-index', slotId);
 
+      if (this.element.getAttribute('data-multi-size')
+          && Object.keys(getAllParentAdContainers(this.element)).length != 0) {
```

Comment:
If we have a slot with width W and height H, and multi-size size WxH, then the ad request will look like sz=WxH|WxH. If the slot is in a container, then the request will look like sz=WxH. It shouldn't make a difference. Similarly, if the multi-size attribute is invalid, we ignore it anyway, so unsetting it shouldn't matter.

That said, I can understand your objection. I'll try and pull up some numbers.